ls1intum/Artemis
# `Lectures`: Fix lecture unit names not trimmed

### Checklist
#### General
#### Server
### Motivation and Context
Closes #5839 

### Description
Lecture unit names are now trimmed, whenever the name is set.

### Steps for Testing
Prerequisites:
- 1 Instructor
- 1 Course where you can create lectures and lecture units.

1. Log in to Artemis
2. Navigate to Course Administration
3. Navigate to a course
4. Create a lecture
5. Create all kinds of lecture units with a name containing leading and/or trailing spaces.
6. Try to delete all lecture units again. Just type the name without any additional leading or trailing whitespace. Once the name is typed, the delete button should be enabled.
### Review Progress

#### Code Review
#### Manual Tests
### Test Coverage
